sqlpage/templates/form.handlebars

```diff
@@ -3,80 +3,103 @@
         {{#if title}}
             <h2 class="text-center mb-3">{{title}}</h2>
         {{/if}}
-        {{#each_row}}
-            {{#if (or (eq type "radio") (eq type "checkbox"))}}
-                <div class="form-selectgroup form-selectgroup-boxes d-flex flex-column my-1">
-                    <label class="form-selectgroup-item flex-fill">
-                        <input type="{{type}}" name="{{name}}" value="{{value}}" {{#if required}}required{{/if}} {{#if checked}}checked{{/if}} class="form-selectgroup-input">
-                        <div class="form-selectgroup-label d-flex align-items-center p-3">
-                            <div class="me-3">
-                                <span class="form-selectgroup-check"></span>
+        <div class="row">
+            {{#each_row}}
+            <div class="col-md-6 mb-2" >
```

Comment:
I don't think we want to hardcode the `col-md-6` here. This would break existing forms. What I think we want is a new `columns` numeric row-level attribute that controls this CSS property.